applications/ocelloids_xcm_monitoring_service.md

```diff
@@ -0,0 +1,270 @@
+# Ocelloids XCM Transfer Monitoring Service
+
+- **Team Name:** SO/DA zone
+- **Payment Address:** Fiat 04.05.2023, 16:37
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview
+
+This grant proposal is a follow-up to the Ocelloids Monitoring SDK, previously delivered and available here: https://github.com/w3f/Grant-Milestone-Delivery/pull/934
+
+The objective of this grant is to develop an open-source monitoring service using the Ocelloids SDK. This service will monitor XCM transfers across Asset Hub and selected parachains. The primary purpose is to offer exchanges, custodians and other institutions on Asset Hub an efficient means of tracking asset transfers across different parachains. Additionally, we will abstract the underlying pallets responsible for initiating XCM transfers and managing assets, which varies for each parachain. Users will have access to an HTTP API to subscribe to XCM transfers and manage their subscriptions. A Docker image will be provided to facilitate service deployment.
+
+## Project Details
+
+This service is designed for use alongside an Asset Hub node deployment and the Asset Transfer API. The goal is to provide exchanges, custodians and institutions with an easily deployable and manageable monitoring solution for tracking their asset flows across parachains. The service will also be available for any user interested in monitoring XCM transfers between Asset Hub and parachains.
```

Comment:
You use "XCM transfers" a lot. Can you define it? For example, it can mean either burning assets locally and sending instructions to mint them remotely, or can mean moving assets into a reserve account.

Reply:
We have updated the application to specify that it is only for reserve-based transfers.